nttt/arguments.py

```diff
@@ -0,0 +1,130 @@
+from .constants import Constants
+import os
+from pathlib import Path
+
+def get_absolute_path(folder):
+    '''Returns the absolute path for the given folder. Trailing path separators and double quotes are also removed.'''
+
+    folder = folder.strip().rstrip(os.pathsep).rstrip('"')
+    folder = Path(folder).absolute()
+    return folder
+
+def get_step_file(folder, step):
+    '''Returns the step file for the given step number.'''
+
+    return Path(folder, str(step))
```

Comment:
Is this correct? Shouldn't step file path include "step_" part?

Reply:
Good catch!